routing/routes.go

```diff
@@ -266,13 +285,13 @@ func NewRoutes(
 		//
 		// responses:
 		//      500: internalError
-		//      200: statusInfo
+		//      200: disabledFeaturesResponse
 		{
-			"Crippled",
+			"DisabledFeatures",
 			log.ConfigLogName,
 			"GET",
-			"/api/crippled",
-			handlers.CrippledFeatures(conf, prom),
+			"/api/internal/features/disabled",
+			handlers.DisabledFeaturesHandler(conf, prom),
```

Comment:
Adding an `internal` endpoint is a little confusing to me since it's not clear what all should go under there. `/api/crippled` _is_ used by the UI for example. Maybe `/api/metrics` since both these endpoints seem to be metrics related?